core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/remove.kt

```diff
@@ -10,15 +10,44 @@ import kotlin.reflect.KProperty
 
 // region DataFrame
 
-public fun <T> DataFrame<T>.remove(columns: ColumnsSelector<T, *>): DataFrame<T> = removeImpl(allowMissingColumns = true, columns = columns).df
+// region remove
+
+public fun <T> DataFrame<T>.remove(columns: ColumnsSelector<T, *>): DataFrame<T> =
+    removeImpl(allowMissingColumns = true, columns = columns).df
+
 public fun <T> DataFrame<T>.remove(vararg columns: KProperty<*>): DataFrame<T> = remove { columns.toColumns() }
+
 public fun <T> DataFrame<T>.remove(vararg columns: String): DataFrame<T> = remove { columns.toColumns() }
+
 public fun <T> DataFrame<T>.remove(vararg columns: AnyColumnReference): DataFrame<T> = remove { columns.toColumns() }
-public fun <T> DataFrame<T>.remove(columns: Iterable<AnyColumnReference>): DataFrame<T> = remove { columns.toColumnSet() }
+
+@Deprecated(
+    message = "It will be removed in the 0.10.0 release",
+    level = DeprecationLevel.WARNING,
+    replaceWith = ReplaceWith("remove(columns)")
```

Comment:
and `remove(columns)` is what we're going to remove. We still need to decide what to replace it with but it will probably be `remove { columns.toColumnSet() }`